docs/services/navigation/_index.md

```diff
@@ -135,15 +141,14 @@ Go to your robot's **Code Sample** tab on the [Viam app](https://app.viam.com) f
 
 ### Mode
```

Comment:
Hmm, perhaps for engineering: I think `Mode` as get() function is misleading, feels conceptual when talking about both get() and set() tasks. I might suggest `getMode` for the fetching to match `setMode` for the setting.

No action in docs -- exact match to literal function nomenclature is all we can do! -- but I'll consider an eng ticket or feedback or such. Do you feel the same or am I totally off-base here?

Reply:
def an ask for eng but a worthy one-- Most of the get endpoints have been named in this convention, so I assumed it was what they wanted with Go.

Reply:
Thanks! I poked around and indeed this appear to follow our existing go naming conventions, so I'll leave it alone. Thanks for the helpful context!